tensorboard/components/tf_line_chart_data_loader/data-loader-behavior.ts

```diff
@@ -120,7 +119,8 @@ export const DataLoaderBehavior = {
 
     if (!this.isAttached) return;
     this._loadDataAsync = this.async(() => {
-      this.dataLoading = true;
+      // Using a setter does not notify the impl observers.
```

Comment:
Hmm, this is basically the same issue as the loadKey issue, right?  So it seems like maybe the behavior observer/property connection is buggy or broken somehow.